src/org/zaproxy/zap/extension/brk/ExtensionBreak.java

```diff
@@ -125,13 +132,21 @@ public void hook(ExtensionHook extensionHook) {
 	    super.hook(extensionHook);
 	    
 	    extensionHook.addOptionsParamSet(getOptionsParam());
+        extensionHook.addProxyListener(getProxyListenerBreak());
+        extensionHook.addSessionListener(this);
+        extensionHook.addOptionsChangedListener(this);
+
+        API.getInstance().registerApiImplementor(api);
 	    
 	    if (getView() != null) {
-	        breakpointMessageHandler = new BreakpointMessageHandler(getBreakPanel());
+		    breakPanel = new BreakPanel(this, getOptionsParam());
+		    breakPanel.setName(Constant.messages.getString("tab.break"));
+	        breakpointMessageHandler = new BreakpointMessageHandler2(breakPanel);

```

Comment:
Seems to be something going on with indentation here? The previous two lines are over 1 space from this one...